packages/nextjs/src/utils/config.ts

```diff
@@ -175,3 +196,23 @@ export function withSentryConfig(
     webpack: newWebpackExport,
   };
 }
+
+/**
+ * Store the future location of the webpack-processed version of `sentry.server.config.js` in a runtime env variable, so
+ * we know where to find it when we want to initialize the SDK.
+ *
+ * @param config The webpack config object for this build
+ */
+function memoizeOutputPath(config: WebpackConfig): void {
```

Comment:
Nit: in my eyes memoize === storing result of expensive function calls in a cache data structure. I don’t think this func captures that exactly. Maybe rename to something like `setSentryEnvVar`, to better capture what it is doing?

Reply:
That's a fair point. I think I was trying to capture the why/what (stash it somewhere for later use) rather than the how (stick it in an env variable, which is sort of an implementation detail). I'm not against `setEnvVars` (there will eventually likely be more than one) or some such, but if you were coming up with a "why/what" verb, what do you think would be a good alternative?

Reply:
I see. IMO I feel like it's good for this function to be a little explicit about it's implementation. Setting an env variable is a side effect that should be highlighted to a reader. For example, my first glance reading something like `storeWepPackOutputPath()` is that it's setting something in memory, not to the file system.